src/content/docs/browser/new-relic-browser/browser-pro-features/browser-error-profiles-find-error-causes.mdx

```diff
@@ -29,7 +29,7 @@ The screenshot below shows an example of an error profile.
 <img
   title="jserror_agentos.png"
   alt="jserror_agentos.png"
-  src={jserrorAgentos}
+  src={browserErrorProfileExample1}
```

Comment:
@a-sassman it looks like this should be `browserErrorProfileExample21`

Reply:
I fixed this with the latest commit.